Formula/msgpack-cxx.rb

```diff
@@ -0,0 +1,57 @@
+class MsgpackCxx < Formula
+  desc "MessagePack implementation for C++ / msgpack.org[C++]"
+  homepage "https://msgpack.org/"
+  url "https://github.com/msgpack/msgpack-c/releases/download/cpp-4.0.1/msgpack-cxx-4.0.1.tar.gz"
+  sha256 "acba76ebc3ff9af2fd2649f319b25cd2387c66f75ae74a7a652fe3386e45d498"
+  license "BSL-1.0"
+  head "https://github.com/msgpack/msgpack-c.git", branch: "cpp_master"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+
+    # fix include conflicts between msgpack and msgpack-cxx
+    include.install include/"msgpack" => "msgpack-cxx"
+    include.install include/"msgpack.hpp" => "msgpack-cxx.hpp"
+    inreplace include/"msgpack-cxx.hpp", "\"msgpack/", "\"msgpack-cxx/"
+  end
+
+  test do
+    # Reference: https://github.com/msgpack/msgpack-c/blob/cpp_master/QUICKSTART-CPP.md
+    (testpath/"test.cpp").write <<~EOS
+      #include <msgpack.hpp>
```

Comment:
```suggestion
      #include <msgpack-cxx.hpp>
```

Reply:
With that change, this is the error in my local
```
In file included from test.cpp:1:
In file included from /usr/local/Cellar/msgpack-cxx/4.0.1/include/msgpack-cxx.hpp:10:
/usr/local/Cellar/msgpack-cxx/4.0.1/include/msgpack-cxx/object.hpp:13:10: fatal error: 'msgpack/object_decl.hpp' file not found
#include "msgpack/object_decl.hpp"
         ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
Error: msgpack-cxx: failed
```